assets/quick-order-list.js

```diff
@@ -79,13 +79,15 @@ class QuickOrderList extends HTMLElement {
         type: `${this.stickyHeaderElement.getAttribute('data-sticky-type')}`
       };
     }
+    this.totalBar = this.querySelector('.quick-order-list__total');
+    if (this.totalBar) {
+      this.totalBarPosition = window.innerHeight - this.totalBar.offsetHeight;
```

Comment:
I think it is worth updating the code on line 235 to`const totalBarCrossesInput = this.totalBarPosition ? inputBottomBorder > this.totalBarPosition : false`.

Reply:
Makes sense. Best solution is where we don't have completely different logic running based on the array length 1 v/s many but it is okay for now. We can look at improving this in the future.